### PR TITLE
[ZH] Document `!mp ban` command

### PR DESCRIPTION
**Pull request by ChickenGoldSS**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

## Changed Files

+ Added translation about "!mp ban" command in zh.md

### CODE REVIEW

**Review by YumeMuzi — APPROVED** [review]

File: wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands/zh.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ outdated_since: a4e45cf0738ff0ce2327cba8b9efe89d53e1ef9f
   - 倒计时将会在距离结束 30 秒，10 秒，5 秒时公告。
 - `!mp aborttimer` - 结束倒计时 (普通倒计时和比赛倒计时)。
 - `!mp kick <username>` - 踢出指定的玩家。
+- `!mp ban <username>` - 禁用指定的玩家。
```
Comment (CloneWith) [inline wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands/zh.md:39]: “禁用”玩家，这句话不通。
我没用过这个命令，但是从字面上来看类似于“禁止玩家进入房间”。

**Review by CloneWith — CHANGES_REQUESTED** [review]

File: wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands/zh.md (hunk 1 above)
Comment (ChickenGoldSS) [reply wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands/zh.md]: 已修改为“禁止指定的玩家进入房间”。

**Review by CloneWith — APPROVED** [review]

**Comment by cl8n** [comment]
this is missing some changes still (see <https://osu.wiki/status/diff/a4e45cf0738ff0ce2327cba8b9efe89d53e1ef9f/wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands/en.md>)

**Comment by cl8n** [comment]
actually I guess it's fine since you're keeping the outdated marker.

**Review by cl8n — APPROVED** [review]

**Pull request merged**